<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login - Spire</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#LoginDialog {
  background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FFFFFF, stop: 1 #DCDCDC);
  border-style: outset;
  border-radius: 3px;
  border-width: 2px;
  border-color: #686868;
}
</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="m_cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_loginTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Reference Sans Serif</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#m_loginTitle {
  color: rgb(51, 102, 153);
}
</string>
   </property>
   <property name="text">
    <string>Spire</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_loginButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QWidget" name="m_credentialsBox" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>350</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#m_credentialsBox {
    background-color: #D2E2F2;
    border-style: solid;
    border-radius: 3px;
    border-width: 2px;
    border-color: #89ADE4;
}
</string>
   </property>
   <widget class="QWidget" name="m_passwordBox" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>311</width>
      <height>35</height>
     </rect>
    </property>
    <widget class="QLabel" name="m_passwordLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QWidget" name="m_passwordInputBox" native="true">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>7</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>21</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#m_passwordInputBox {
  border-style: solid;
  border-width: 2px;
  border-color: #89ADE4;
}
</string>
     </property>
     <widget class="QLineEdit" name="m_passwordInput">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>147</width>
        <height>17</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>147</width>
        <height>17</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>147</width>
        <height>17</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#m_passwordInput {
  border-style: solid;
  border-width: 2px;
  border-color: #FFFFFF;
}
</string>
      </property>
      <property name="maxLength">
       <number>12</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QWidget" name="m_chromaHash1" native="true">
      <property name="geometry">
       <rect>
        <x>149</x>
        <y>2</y>
        <width>10</width>
        <height>17</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="m_chromaHash2" native="true">
      <property name="geometry">
       <rect>
        <x>159</x>
        <y>2</y>
        <width>10</width>
        <height>17</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="m_chromaHash3" native="true">
      <property name="geometry">
       <rect>
        <x>169</x>
        <y>2</y>
        <width>10</width>
        <height>17</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="m_usernameBox" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>311</width>
      <height>35</height>
     </rect>
    </property>
    <widget class="QLabel" name="m_usernameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_usernameInput">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>7</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit#m_usernameInput {
  border-style: solid;
  border-width: 2px;
  border-color: #89ADE4;
}
</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="maxLength">
      <number>24</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="m_resultLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>11</green>
          <blue>15</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>11</green>
          <blue>15</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>106</red>
          <green>104</green>
          <blue>100</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="m_serverBox" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>311</width>
      <height>35</height>
     </rect>
    </property>
    <widget class="QLabel" name="m_serverLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Server:</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>183</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="m_serverComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>181</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>181</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_usernameInput</tabstop>
  <tabstop>m_passwordInput</tabstop>
  <tabstop>m_serverComboBox</tabstop>
  <tabstop>m_loginButton</tabstop>
  <tabstop>m_cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
